doc/source/ray-observability/user-guides/debug-apps/optimize-performance.rst

```diff
@@ -347,6 +347,21 @@ Here are the steps to use PyTorch Profiler during training with Ray Train or bat
 * Visualize the results with tools like Tensorboard.
 
 
+GPU Profiling with Nsight Profiler
+----------------------------------
+Here are the steps to use Nsight Profiler with Ray Core:
```

Comment:
I think all this info should be just inside the original section. You can simply add 

GPU profiling is critical for ML training and inference. Ray allows users to run Nsight profiler with Ray actors and tasks. See <direct link to the section> for details.